frontend/mr-admin-flate/src/api/avtaler/useAvtale.ts

```diff
@@ -0,0 +1,15 @@
+import { useQuery } from "@tanstack/react-query";
+import { useParams } from "react-router-dom";
+import { mulighetsrommetClient } from "../clients";
+import { QueryKeys } from "../QueryKeys";
+
+export function useAvtale() {
+  const { avtaleId } = useParams<{ avtaleId: string }>();
+  if (!avtaleId) {
+    throw new Error("Fant ingen avtaleId i url");
```

Comment:
Hadde vært praktisk med en error boundary på et lurt sted i app'en :ok_hand:

Reply:
Jepp, kommer her https://trello.com/c/tozAZroD/616-opprett-errorboundaries-for-admin-flate